account_companyweb/wizard/account_companyweb_wizard.py

```diff
@@ -47,9 +47,10 @@ class account_companyweb_wizard(orm.TransientModel):
         'result': fields.float('Fiscal Year Profit/Loss (+/-)', readonly=True),
     }
 
-    def get_update_values(self, cr, uid, ids, wizard, context=None):
+    def get_update_values(self, cr, uid, ids, context=None):
```

Comment:
Browse records can not be passed over xmlrpc when calling from scripts; the parameter ids provides sufficient information to retrieve the concerned wizard object.